controllers/events.js

```diff
@@ -193,11 +193,38 @@ const endActiveEvent = async (req, res) => {
   }
 };
 
+/**
+ * Adds a peer to an event.
+ *
+ * @async
+ * @function
+ * @param {Object} req - The Express request object.
+ * @param {Object} res - The Express response object.
+ * @returns {Promise<Object>} The JSON response with the added peer data and a success message.
+ * @throws {Object} The JSON response with an error message if an error occurred while adding the peer.
+ */
+const addPeerToEvent = async (req, res) => {
+  try {
+    const data = await eventQuery.addPeerToEvent(req.body);
+    return res.status(200).json({
+      data,
+      message: `Peer is added to the event`,
+    });
+  } catch (error) {
+    logger.error({ error });
+    return res.status(500).json({
```

Comment:
Please use `res.boom.badImplementation` for consistency

Reply:
Yes, fixed it. Thank you